maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/booterclient/JarManifestForkConfiguration.java

```diff
@@ -103,7 +103,7 @@ private File createJar( @Nonnull List<String> classPath, @Nonnull String startCl
         {
             file.deleteOnExit();
         }
-        Path parent = file.getParentFile().toPath();
+        Path parent = file.getParentFile().toPath().normalize();
```

Comment:
It would be worth to copy [1] and integrate it in `surefire-its` as an integration test.
[1]: https://github.com/daniel-kurzynski/reproduce-surefire-issue

Reply:
I will guide you on how to write an IT in the evening.
Meanwhile I would like to see the path. Did you grab the path from debugger?
I have used another project and the path was `d:\vcs\release\.\surefire-1605\target\classes` in my case but the project works without the fix.